src/game/ui/BuildMenu.cxx

```diff
@@ -8,6 +8,9 @@
 #include <Settings.hxx>
 #include <GameStates.hxx>
 #include <MapLayers.hxx>
+#include <TileManager.hxx>
```

Comment:
Again, it looks like these are the only changes you made to this file, so there's no new lines that actually use these include statements.